migrate/hash/action.yml

```diff
@@ -0,0 +1,27 @@
+# Generated by go run ./cmd/gen github-manifest; DO NOT EDIT.
+name: Migrate Hash
```

Comment:
@ronenlu  You need add the new action in https://github.com/ariga/atlas-action/blob/master/atlasaction/manifest.yml instead